src/utils.py

```diff
@@ -248,6 +248,7 @@ def _charm_config_to_datadict(self):
             "log_level": "INFO",
             "output_dir": self.charm_config["backup-dir"],
             "timeout": self.charm_config["timeout"],
+            "app_backup_basedir": self.charm_config["backup-basedir"],
```

Comment:
Please take a look in my [comment](https://github.com/canonical/juju-backup-all/pull/50/files#r1662533040) because it seems that you have different names for the same thing.

Reply:
Updated